pig/src/main/java/org/jboss/pnc/bacon/pig/impl/config/RepoGenerationData.java

```diff
@@ -44,6 +44,8 @@
     private boolean includeJavadoc;
     private boolean includeLicenses;
     private boolean includeMavenMetadata;
+    private boolean includeOffliner;
+    private String offlinerManifest;
```

Comment:
Could we name it `offlineManifestFilename` or something more descriptive?

Reply:
@thescouser89 Re documentation - this links back to my point about gh-pages - then a separate linked PR could update the docs. I _really_ think this should be considered so new features such as this can be documented in PRs as well - we've made this mandatory for pme/gme for instance.